spec/unit/pluginHandlers/common.spec.js

```diff
@@ -35,38 +36,39 @@ var deleteJava = common.__get__('deleteJava');
 var copyNewFile = common.__get__('copyNewFile');
 
 describe('common platform handler', function () {
+    afterEach(() => {
+        fs.removeSync(test_dir);
+        fs.removeSync(non_plugin_file);
+    });
+
     describe('copyFile', function () {
         it('Test#001 : should throw if source path not found', function () {
-            fs.removeSync(src);
             expect(function () { copyFile(test_dir, src, project_dir, dest); })
                 .toThrow(new Error('"' + src + '" not found!'));
         });
 
         it('Test#002 : should throw if src not in plugin directory', function () {
             fs.ensureDirSync(project_dir);
-            fs.writeFileSync(non_plugin_file, 'contents', 'utf-8');
+            fs.outputFileSync(non_plugin_file, 'contents');
             var outside_file = '../non_plugin_file';
             expect(function () { copyFile(test_dir, outside_file, project_dir, dest); })
                 .toThrow(new Error('File "' + path.resolve(test_dir, outside_file) + '" is located outside the plugin directory "' + test_dir + '"'));
-            fs.removeSync(test_dir);
         });
 
         it('Test#003 : should allow symlink src, if inside plugin', function () {
-            fs.ensureDirSync(java_dir);
-            fs.writeFileSync(java_file, 'contents', 'utf-8');
+            fs.outputFileSync(java_file, 'contents');
 
             // This will fail on windows if not admin - ignore the error in that case.
             if (ignoreEPERMonWin32(java_file, symlink_file)) {
                 return;
             }
 
             copyFile(test_dir, symlink_file, project_dir, dest);
-            fs.removeSync(project_dir);
```

Comment:
Won't this leave traces of the test behind? It's not cleared via the afterEach hook.

Reply:
No. `project_dir` is contained in `test_dir` which is removed in `afterEach`.